samples/gltf_viewer.cpp

```diff
@@ -907,6 +915,13 @@ int main(int argc, char** argv) {
         Skybox* skybox = scene->getSkybox();
         applySettings(engine, app.viewer->getSettings().viewer, &camera, skybox, renderer);
 
+        // technically we don't need to do this each frame
+        auto& tcm = engine->getTransformManager();
+        TransformManager::Instance const& root = tcm.getInstance(app.rootTransformEntity);
+        tcm.setParent(tcm.getInstance(camera.getEntity()), root);
```

Comment:
Does this work if the user selects a glTF camera?

Reply:
yeah it works.